### PR TITLE
Style submenus

### PR DESCRIPTION
**Pull request by johnBartos**
### This PR will...
- Style the submenus
- Add accessibility attributes

### Why is this Pull Request needed?
We want the submenu to look good 😎

### Are there any points in the code the reviewer needs to double check?
No

### Are there any Pull Requests open in other repos which need to be merged with this?
No

#### Addresses Issue(s):

JW8-317

### CODE REVIEW

**Comment by egreaves** [comment]
@johnBartos the quality sub-menu is showing when there's only one quality.
screen shot 2017-08-20 at 10 46 50 pm

File: src/js/view/controls/controls.js [hunk 1]
```diff
@@ -344,11 +346,10 @@ export default class Controls {
...
         const settingsMenu = this.settingsMenu;
 
-        model.change('mediaModel', function(newModel, mediaModel) {
-
+        model.change('mediaModel', (newModel, mediaModel) => {
             // Quality Levels
-            mediaModel.on('change:levels', function (changedModel, levels) {
-                if (!levels) {
+            mediaModel.on('change:levels', (changedModel, levels) => {
+                if (!levels || levels && levels.length <= 1) {
```
Comment (egreaves) [inline src/js/view/controls/controls.js]: This is the same thing as `!levels || levels.length <= 1`

File: src/js/view/controls/controls.js [hunk 2]
```diff
@@ -361,16 +362,16 @@ export default class Controls {
...
                 if (qualitiesSubmenu) {
                     qualitiesSubmenu.activateItem(currentQualities);
                 }
             });
 
             // Audio Tracks
-            mediaModel.on('change:audioTracks', function (changedModel, audioTracks) {
-                if (!audioTracks) {
+            const onAudiotracksChange = (changedModel, audioTracks) => {
+                if (!audioTracks || audioTracks && audioTracks.length <= 1) {
```
Comment (egreaves) [inline src/js/view/controls/controls.js]: `!audioTracks || audioTracks.length <= 1`

File: src/js/view/controls/controls.js [hunk 3]
```diff
@@ -381,22 +382,25 @@ export default class Controls {
...
             });
+            // change:audioTracks does not get triggered if the next item has no tracks, so trigger it every time the mediaModel changes (i.e. we're on a new item)
+            onAudiotracksChange(newModel, model.get('audioTracks'));
         });
 
         // Captions
         model.change('captionsList', (changedModel, captionsList) => {
             const controlbarButton = controlbar.elements.captionsButton;
-            if (!captionsList) {
+            if (!captionsList || captionsList && captionsList.length <= 1) {
```
Comment (egreaves) [inline src/js/view/controls/controls.js]: `!captionsList || captionsList.length <= 1`

**Comment by hghazzi** [comment]
test this please

**Pull request merged**